docs/docs/Agents/agent-tool-calling-agent-component.md

```diff
@@ -181,7 +177,7 @@ inputs = [
 
 ## Add flows as tools
 
-An agent can use flows that are saved in your workspace as tools.
+An agent can use flows that are saved in your workspace as tools with the [flow as tool](/components-logic#flow-as-tool) component.
```

Comment:
How is this capitalized in the UI? You don't have it capitalized here, but you have initial caps on it in the next file.